bundles/org.openhab.ui/web/src/pages/settings/things/things-list.vue

```diff
@@ -210,6 +225,7 @@ export default {
           searchbar.search(filterQuery)
         }
         if (groupBy === 'alphabetical') this.$refs.listIndex.update()
+        if (groupBy === 'state') this.$refs.listIndex.update()
```

Comment:
This is probably not necessary since you're not using a list index when you're grouping by state.

Reply:
```suggestion

```